src/cascadia/TerminalApp/TerminalSettings.cpp

```diff
@@ -6,8 +6,202 @@
+
+        if (!profile.StartingDirectory().empty())
+        {
+            _StartingDirectory = profile.GetEvaluatedStartingDirectory();
+        }
+
+        // GH#2373: Use the tabTitle as the starting title if it exists, otherwise
+        // use the profile name
+        _StartingTitle = !profile.TabTitle().empty() ? profile.TabTitle() : profile.Name();
+
+        if (profile.SuppressApplicationTitle())
+        {
+            _SuppressApplicationTitle = profile.SuppressApplicationTitle();
+        }
+
+        if (!profile.ColorSchemeName().empty())
+        {
+            ApplyColorScheme(profile.ColorSchemeName(), schemes);
+        }
+        if (profile.Foreground())
+        {
+            _DefaultForeground = til::color{ profile.Foreground().Value() };
+        }
+        if (profile.Background())
+        {
+            _DefaultBackground = til::color{ profile.Background().Value() };
+        }
+        if (profile.SelectionBackground())
+        {
+            _SelectionBackground = til::color{ profile.SelectionBackground().Value() };
+        }
+        if (profile.CursorColor())
+        {
+            _CursorColor = til::color{ profile.CursorColor().Value() };
+        }
+
+        _ScrollState = profile.ScrollState();
+
+        if (!profile.BackgroundImagePath().empty())
+        {
+            _BackgroundImage = profile.GetExpandedBackgroundImagePath();
+        }
+
+        _BackgroundImageOpacity = profile.BackgroundImageOpacity();
+        _BackgroundImageStretchMode = profile.BackgroundImageStretchMode();
+
+        _BackgroundImageHorizontalAlignment = profile.BackgroundImageHorizontalAlignment();
+        _BackgroundImageVerticalAlignment = profile.BackgroundImageVerticalAlignment();
+
+        _RetroTerminalEffect = profile.RetroTerminalEffect();
+
+        _AntialiasingMode = profile.AntialiasingMode();
+
+        if (profile.TabColor())
+        {
+            const til::color colorRef{ profile.TabColor().Value() };
+            _TabColor = static_cast<uint32_t>(colorRef);
+        }
+    }
+
+    // Method Description:
+    // - Applies appropriate settings from the globals into the given TerminalSettings.
+    // Arguments:
+    // - settings: a TerminalSettings object to add global property values to.
+    // Return Value:
+    // - <none>
+    void TerminalSettings::_ApplyGlobalSettings(const TerminalApp::GlobalAppSettings& globalSettings) noexcept
+    {
+        _InitialRows = globalSettings.InitialRows();
+        _InitialCols = globalSettings.InitialCols();
+
+        _WordDelimiters = globalSettings.WordDelimiters();
+        _CopyOnSelect = globalSettings.CopyOnSelect();
+        _ForceFullRepaintRendering = globalSettings.ForceFullRepaintRendering();
+        _SoftwareRendering = globalSettings.SoftwareRendering();
+        _ForceVTInput = globalSettings.ForceVTInput();
+    }
+
+    // Method Description:
+    // - Apply our values to the given TerminalSettings object. Sets the foreground,

```

Comment:
update comments for the functions in `TerminalSettings`, this one and a couple above this.